test/acs-engine-test/main.go

```diff
@@ -380,6 +425,9 @@ func mainInternal() error {
 		fmt.Println("Warning: BUILD_NUMBER is not set or invalid. Assuming 0")
 		buildNum = 0
 	}
+	if os.Getenv("SKIP_METRICS") == "y" {
```

Comment:
it is an environment variable to control test flow. It might be set in a Jenkins job.